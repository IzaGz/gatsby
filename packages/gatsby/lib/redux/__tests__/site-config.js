import { store } from "../index.js";

describe(`add site config`, () => {
  it(`allow you to add basic site config`, () => {
    const config = {
      siteMetadata: {
        title: "yo testing",
      },
      plugins: [],
    };
    store.dispatch({
      type: "SET_SITE_CONFIG",
      payload: config,
    });
    expect(store.getState()).toMatchSnapshot();
  });

  it(`Validates configs with unsupported options`, () => {
    const config = {
      someRandomThing: "hi people",
      plugins: [],
    };
    store.dispatch({
      type: "SET_SITE_CONFIG",
      payload: config,
    });
    expect(store.getState()).toMatchSnapshot();
  });
});
