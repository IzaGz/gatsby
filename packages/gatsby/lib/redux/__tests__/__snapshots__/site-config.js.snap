exports[`add site config allow you to add basic site config 1`] = `
Object {
  "config": Object {
    "plugins": Array [],
    "siteMetadata": Object {
      "title": "yo testing",
    },
  },
  "flattenedPlugins": Array [],
  "nodes": Object {},
  "pages": Array [],
  "plugins": Array [],
  "program": Object {},
}
`;
